app/containers/UIKit/VideoConferenceBlock/components/styles.ts

```diff
@@ -0,0 +1,126 @@
+import { StyleSheet } from 'react-native';
+
+import { useTheme } from '../../../../theme';
+import sharedStyles from '../../../../views/Styles';
+
+export default function useStyle() {
+	const { colors } = useTheme();
+	return StyleSheet.create({
+		container: { height: 108, flex: 1, borderWidth: 1, borderRadius: 2, marginTop: 8, borderColor: colors.conferenceCallBorder },
```

Comment:
Recently we increased most of the borderRadius to 4.
We can confirm with design team, but I think Figma might be outdated and we need to use 4 here already and on some other places below